src/plugins/auto_batch/src/remote_context.hpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#pragma once
+
+#include <memory>
+
+#include "openvino/runtime/iremote_context.hpp"
+#include "openvino/runtime/iremote_tensor.hpp"
+#include "openvino/runtime/remote_context.hpp"
```

Comment:
looks like you don't deal with IRemotecontext？ if yes, then remove redundant headers

Reply:
Will remove